src/connectedvmware/azext_connectedvmware/tests/latest/test_arcvmware_scenario.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+from knack.util import CLIError
+from azure.cli.testsdk import ScenarioTest
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class ConnectedvmwareScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_connectedvmware')
+    def test_connectedvmware_vcenter(self, resource_group):
+
+        self.kwargs.update({'name': 'test1'})
+
+        # raise CLIError('TODO')
```

Comment:
Could you upload the yaml file of `test_connectedvmware_vcenter` to fix the CI issue, or remove this test directly

Reply:
I have removed this test file.